Cargo.toml

```diff
@@ -148,7 +148,7 @@ suppaftp = { version = "4.5", default-features = false, features = [
   "async-rustls",
 ], optional = true }
 time = { version = "0.3", features = ["serde"] }
-tokio = { version = "1.20", features = ["fs"] }
+tokio = { version = "1.20", features = ["fs", "io-util"] }
```

Comment:
Is `io-util` required?

Reply:
No, it's not necessary. The feature has been removed.